lib/avtp_pipeline/acmp/openavb_acmp_sm_listener.c

```diff
@@ -951,7 +963,8 @@ void openavbAcmpSMListenerSet_talkerTestFastConnect(
 				openavbAcmpSMListenerSet_doFastConnect(pDescriptor->stream, flags, talker_unique_id, talker_entity_id, controller_entity_id);
 			}
 			else {
-				AVB_LOGF_ERROR("Unexpected fast connect info for talker_entity_id=" ENTITYID_FORMAT, ENTITYID_ARGS(entity_id));
+				AVB_LOGF_DEBUG("Fast connect info for talker_entity_id=" ENTITYID_FORMAT " no longer valid", ENTITYID_ARGS(entity_id));
+				memset(pDescriptor->fast_connect_talker_entity_id, 0, 8);

```

Comment:
I believe this is OK and that fast_connect_talker_entity_id is 8 bytes, however, consider using sizeof() here.